prow/cmd/deck/main.go

```diff
@@ -1338,6 +1333,11 @@ func handleRerun(prowJobClient prowv1.ProwJobInterface, createProwJob bool, cfg
 					l.WithError(err).Errorf("Error checking if user can trigger job")
 					return
 				}
+				logrus.WithFields(logrus.Fields{
+					"user":    login,
+					"job":     newPJ.Spec.Job,
```

Comment:
Job names are not unique :-/

Reply:
I think that's fine for my use case. I'm mainly interested in seeing how people generally use or try to use the rerun button.